frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx

```diff
@@ -180,14 +189,17 @@ function OutputForDocModal({
       setRows([]);
       return;
     }
+    let url = `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/prompt-output/?tool_id=${details?.tool_id}&prompt_id=${promptId}&profile_manager=${profile}&is_single_pass_extract=${singlePassExtractMode}`;
```

Comment:
This can be const if we are not modifying it below.

Reply:
We will be changing it in #L194, if it is a public source.